src/electos/ballotmaker/ballots/build_ballots.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from datetime import datetime
+from pathlib import Path
+
+from electos.ballotmaker.ballots.ballot_layout import build_ballot
+from electos.ballotmaker.constants import PROGRAM_NAME
+from electos.ballotmaker.data.models import ElectionData
+
+logging.getLogger(__name__)
+
+
+def get_election_header(election: ElectionData) -> dict:
+    """extract the shared data for ballot headers"""
+    name = election.name
+    end_date = election.end_date
+    election_type = election.type
+    return {
+        "Name": name,
+        "EndDate": end_date,
+        "Type": election_type,
+    }
+
+
+def build_ballots(election: ElectionData) -> Path:
+
+    # create the directories needed
+    now = datetime.now()
+    date_time = now.strftime("%Y_%m_%dT%H%M%S")
```

Comment:
YMMV, but I found this a little hard to read. Ideally there's a separator between the time as well as the date. _OR_, don't use "T", use a different separator ("+" maybe, or another "_")? The "T" matters for ISO formatted names, but this isn't following that spec.

This doesn't matter for the demo. I'm mostly noting for future reference, from experience scanning large lists of output with timestamps as part of the file name, when tired eyes are glazing over 😫.

Reply:
You've identified some of the oldest code in the repo, and it's looking long in the tooth, time for a rewrite!

Reply:
I marked this with a TODO and a link back to this thread right in the code.